tests/bson/bug0325.phpt

```diff
@@ -16,6 +16,6 @@ var_dump($value);
 ===DONE===
 <?php exit(0); ?>
 --EXPECTF--
-Warning: %s\toPHP(): Reading document did not exhaust input buffer in %s on line %d
+Warning:%sReading document did not exhaust input buffer in %s on line %d%a
```

Comment:
@derickr Can you paste the full error message here for future reference?
Also, is it all messages or just thig particular one?